Test/NotCSharp/Python/EASY-UnivaluedBinaryTree.py

```diff
@@ -0,0 +1,21 @@
+#https://leetcode.com/problems/univalued-binary-tree/
+
+'''
+Definition for a binary tree node.
+class TreeNode:
+    def __init__(self, x):
+    self.val = x
+    self.left = None
+    self.right = None
+'''
+
+class Solution:
```

Comment:
I'll leave it for you to decide which solution makes more sense to you
https://paste.ofcode.org/dXCFkPeabHVZiuWJ8CCdtw
Feel free to pick

Reply:
Thanks - but I think the solution I posted was simple and easy to remember for me. I would like to keep that but am happy to add your option in the file below this? Let me know.